src/midi/midi_fluidsynth.cpp

```diff
@@ -313,6 +313,8 @@ void MidiHandlerFluidsynth::Close()
 	if (!is_open)
 		return;
 
+	HaltSequence();
```

Comment:
I don't think we need this for built-in FluidSynth.

Reply:
(same answer above; I'm inclined to agree, but curious what you think about my reason).